spec/index.html

```diff
@@ -748,7 +773,32 @@ <h2>Literals</h2>
     <p>denote the same <a data-lt="literal value">value</a>, but are not the
       same literal <a>RDF terms</a> and are not
       <a>term-equal</a> because their
-      <a>lexical form</a> differs.</p>
+      <a>lexical forms</a> differ.</p>
+
+    <section id="section-text-direction" class="informative">
+      <h3>Initial Text Direction</h3>
+
+      <p>The <a>base direction</a> of a <a>directional language-tagged string</a>
+        provides a means of establishing the initial direction of text,
+        including text which is mixture of right-to-left and left-to-right scripts.
+        The [[[?UAX9]]] [[?UAX9]] provides support for automatically rendering
+        a sequence of characters in logical order,
+        so that they are visually ordered as expected.
+        However, this is not sufficient to correctly render bidirectional text.</p>
+
+      <p>For example, the text `"פעילות הבינאום, W3C"` with language tag "he" (for Hebrew) may be displayed
+        incorrectly unless the <a>base direction</a> of `rtl` (for right-to-left)
+        is also provided.
+        When this is provided to a user agent,
+        for example using the HTML `dir` attribute,
+        it would be correctly presented as follows: <div lang="he" dir="rtl">פעילות הבינאום, W3C</div></p>
```

Comment:
Do you intend for this to display with a base direction of LTR here so that the later tagged example rendering differently is obvious? Or did you omit the <span dir=rtl> by not observing it?

Note that our specs are translated, so if your intention is the make this show the LTR display for contrast, you might want to do <bdi dir=ltr translate=no> around it.

If you want to show logical order (which won't make sense) would require an `lro` markup, but would have to be explained (by using the code points).

The simplest is probably to say something like:

```suggestion
<p>For example, some text with a language tag "he" might be displayed in a left-to-right context (such as an English web page) as <bdi dir=ltr>פעילות הבינאום, W3C</bdi>, which is incorrect. When provided to a user agent including base direction information (such as using HTML's `dir` attribute) it can then be correctly presented as:</p>
<div lang="he" dir="rtl">פעילות הבינאום, W3C</div>
```